src/parser/jobs/smn/modules/Pets.js

```diff
@@ -190,49 +246,59 @@ export default class Pets extends Module {
+		const missedTicks = this._slipstreams.reduce((acc, cur) => {
+			const tickCount = cur.ticks.length
+			const possibleTicks = Math.min(SLIPSTREAM_TICKS, Math.floor((this.parser.fight.end_time - cur.cast.timestamp) / SLIPSTREAM_TICK_SPEED))
+
+			return acc + (possibleTicks - Math.min(possibleTicks, tickCount))
+		}, 0)
+		if (missedTicks > 0) {
+			this.suggestions.add(new TieredSuggestion({
+				icon: ACTIONS.SLIPSTREAM.icon,
+				tiers: SLIPSTREAM_SEVERITY,
+				value: missedTicks,
+				content: <Trans id="smn.pets.suggestions.slipstream-ticks.content">
+					Ensure you use <ActionLink {...ACTIONS.SLIPSTREAM} /> such that it can deal damage for its entire duration.
+					Summoning another pet or recasting Slipstream will prevent any remaining ticks of a cast.
+				</Trans>,
+				why: <Trans id="smn.pets.suggestions.slipstream-ticks.why">
+					<Plural value={missedTicks} one="# missed tick" other="# missed ticks" /> of Slipstream.
+				</Trans>,
+			}))
+		}
 
-		// Pet suggestions based on party comp
-		// TODO: This does not account for invuln periods
-		const numCasters = roles[ROLES.MAGICAL_RANGED.id]
-		const mostUsedPet = Array.from(this._petUptime.keys()).sort(
-			(a, b) => this._petUptime.get(b) - this._petUptime.get(a)
-		)[0]
+		// Ensure Garuda is being used in AoE
+		if (this._badWindBlades > 0) {
+			const garudaStPercent = (((this._badWindBlades * WIND_BLADE_RECAST) / this._petUptime.get(PETS.GARUDA_EGI.id)) * 100)
 
-		if (numCasters > 1 && mostUsedPet !== PETS.GARUDA_EGI.id) {
-			this.suggestions.add(new Suggestion({
+			this.suggestions.add(new TieredSuggestion({
 				icon: ACTIONS.SUMMON.icon,
-				severity: SEVERITY.MEDIUM,
-				content: <Trans id="smn.pets.suggestions.prefer-garuda.content">
-					You should be primarily using Garuda-Egi when in parties with casters other than yourself - they will benefit from <ActionLink {...ACTIONS.CONTAGION}/>'s Magic Vulnerability Up.
+				tiers: GARUDA_ST_SEVERITY,
+				value: garudaStPercent,
+				content: <Trans id="smn.pets.suggestions.garuda-st.content">
+					Garuda-Egi should only be used for AoE.
 				</Trans>,
-				why: <Trans id="smn.pets.suggestions.prefer-garuda.why">
-					{this.getPetUptimePercent(mostUsedPet)}% {this.getPetName(mostUsedPet)} uptime, Garuda-Egi preferred.
+				why: <Trans id="smn.pets.suggestions.garuda-st.why">
+					Garuda-Egi was attacking a single target {garudaStPercent.toFixed(0)}% of the time it was active.
 				</Trans>,
 			}))
 		}
 
-		// Disabling ifrit check post-4.4 due to changes made to RS
-		if (
-			this.parser.patch.before('4.4') &&
-			numCasters === 1 && mostUsedPet !== PETS.IFRIT_EGI.id
-		) {
+		// Since Ifrit's EA2 and Enkindle hit AoE, we can sometimes tell that
+		// it was used in an AoE situation, but we can't tell any more often than
+		// when those particular skills are used.  Therefore, just list this as Medium
+		// rather than trying to determine a percentage like we did with Garuda.
```

Comment:
I'm a _little_ wary of jumping straight to `MEDIUM` for this - though I can see a level of logic in it, given the CDs on his AoEs are relatively lengthy compared to the summon CDs and such.

Reply:
I wasn't quite sure how to level that either.  On one hand, its possible that there were only multiple targets for a GCD or two around those usages.  On the other hand, every single pet auto attack could theoretically be with multiple targets available and you can't really tell.  Given the primary usage of the site is raids, which have limited multi-target periods, I'd also be fine just marking this minor.  I'm not sure what tiers I'd use if we wanted to tier it.

Reply:
I think it's safe to leave as-is for now - just keep an eye out for feedback, see if there's complaints which might warrant changing severities around.